### PR TITLE
Improved the "in" support

### PR DESCRIPTION
**Pull request by markbates**

### CODE REVIEW

File: postgresql.go [hunk 1]
```diff
@@ -153,20 +152,8 @@ func (p *postgresql) TranslateSQL(sql string) string {
...
-			for _, char := range str {
-				out = append(out, byte(char))
-			}
-			curr++
-		} else {
-			out = append(out, sql[i])
-		}
-	}
-	csql := string(out)
+	csql := sqlx.Rebind(sqlx.DOLLAR, sql)
```
Comment (stanislas-m) [inline postgresql.go]: You might want to do the same for the cockroach driver, since it's a Postgres-compatible DB.

Comment (mclark4386) [reply postgresql.go]: agreed

File: query.go [hunk 2]
```diff
@@ -95,6 +98,15 @@ func (q *Query) Where(stmt string, args ...interface{}) *Query {
...
 		return q
 	}
+	if inRegex.MatchString(stmt) {
+		var inq []string
+		for i := 0; i < len(args); i++ {
+			inq = append(inq, "?")
+		}
+		qs := fmt.Sprintf("(%s)", strings.Join(inq, ","))
+		stmt = strings.Replace(stmt, "(?)", qs, 1)
+		stmt = strings.Replace(stmt, "(?)", qs, 1)
```
Comment (mclark4386) [inline query.go]: Why are you doing this line twice?

Comment (markbates) [reply query.go]: @mclark4386 cause I liked it so much! ;) accidental vim at it's best.

**Comment by markbates** [comment]
@stanislas-m fixed, can you look again when you have a chance? thanks.

**Pull request merged**